client/src/app/main-menu/settings.tsx

```diff
@@ -16,7 +16,11 @@ function Settings({setMenuSection}: Props) {
 	const dispatch = useMessageDispatch()
 	const stats = useSelector(getStats)
 	const settings = useSelector(getSettings)
-	const totalGames = Object.values(stats).reduce((a, b) => a + b, 0)
+
+	const totalGames = Object.values(stats).reduce((a, b) => {
+		if (typeof b === 'number') return a + b
+		return a
+	}, 0)
 
```

Comment:
This is not good. just do `totalGames = stats.w + stats.l + stats.fw + stats.fl + stats.t`

Reply:
true, would be easier to understand